Don't install SeqAn and SeqLib, use separate EB installed versions
Author: Samuel Moors, Vrije Universiteit Brussel (VUB)

diff -ur neusomatic-0.2.1.orig/neusomatic/CMakeLists.txt neusomatic-0.2.1/neusomatic/CMakeLists.txt
--- neusomatic-0.2.1.orig/neusomatic/CMakeLists.txt	2019-07-11 09:04:57.000000000 +0200
+++ neusomatic-0.2.1/neusomatic/CMakeLists.txt	2019-07-24 09:31:22.034020844 +0200
@@ -34,19 +34,12 @@
     message(FATAL_ERROR "GCC/G++ is not found on the system. Please install GCC/G++5.")
 endif()
 
-include (../third_party/seqlib.cmake)
-include (../third_party/seqan.cmake)
 set( CPP_LIBS ${CPP_LIBS} seqlib bwa fml hts bz2)
 set( CPP_LIBS ${CPP_LIBS} dl )
 set( CPP_LIBS ${CPP_LIBS} c)
 
 
 include_directories(${CMAKE_SOURCE_DIR}/include)
-include_directories(../third_party/SeqLib/src/SeqLib )
-include_directories(../third_party/SeqLib/src/SeqLib/htslib )
-include_directories(../third_party/seqan/src/seqan/include)
 
-link_directories(../third_party/lib)
-link_directories(../third_party/SeqLib/src/SeqLib/bin/)
 
 add_subdirectory(cpp)
